merlin-worker/src/main/java/gov/nasa/jpl/aerie/merlin/worker/MerlinWorkerAppDriver.java

```diff
@@ -31,16 +30,14 @@ public static void main(String[] args) throws Exception {
     if (!(store instanceof final PostgresStore postgresStore)) {
       throw new UnexpectedSubtypeError(Store.class, store);
     }
-    final var pgDataSource = new PGDataSource();
-    pgDataSource.setServerName(postgresStore.server());
-    pgDataSource.setPortNumber(postgresStore.port());
-    pgDataSource.setDatabaseName(postgresStore.database());
-    pgDataSource.setApplicationName("Merlin Server");
-
     final var hikariConfig = new HikariConfig();
+    hikariConfig.setDataSourceClassName("org.postgresql.ds.PGSimpleDataSource");
+    hikariConfig.addDataSourceProperty("serverName", postgresStore.server());
+    hikariConfig.addDataSourceProperty("portNumber", postgresStore.port());
+    hikariConfig.addDataSourceProperty("databaseName", postgresStore.database());
+    hikariConfig.addDataSourceProperty("applicationName", "Merlin Server");
     hikariConfig.setUsername(postgresStore.user());
     hikariConfig.setPassword(postgresStore.password());
-    hikariConfig.setDataSource(pgDataSource);
 
     final var hikariDataSource = new HikariDataSource(hikariConfig);
```

Comment:
Yeah. From what I recall, a shutdown hook will work for SIGINT (and [this SO discussion](https://stackoverflow.com/questions/2541475/capture-sigint-in-java) seems to support that), but that doesn't affect SIGKILL.

Either way, I'm resolving this conversation so that it doesn't block this PR.